<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDB</class>
 <widget class="QMainWindow" name="frmDB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>485</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>735</width>
    <height>485</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>735</width>
    <height>485</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>База данных</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtQuery">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск по:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbName">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Имени</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDate">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Дате</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbEvent">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Мероприятию</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbSN">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>150</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер сертификата</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
